### DISCUSSION TITLE
Improve how show version report

### DISCUSSION THREAD

**Discussion by Jean-BaptisteC** [Exodus-Privacy/exodus-android-app#154] [Ideas]
* Actually on 2.0 version, we show [text](https://github.com/Exodus-Privacy/exodus-android-app/blob/51009bed5f8e8e2911d640d8fd16f603443b6280/app/src/main/res/values/strings.xml#L16) to explain if report have same version with installed version.
* To reduce size used for show each app on main page, i propose used icons [(like this PR)](https://github.com/Exodus-Privacy/exodus-android-app/pull/67) to replace old text
* If version report is equal to version installed show this  [icon](https://fonts.google.com/icons?selected=Material%20Icons%20Outlined%3Adone%3A) with green background ![done_black_24dp](https://user-images.githubusercontent.com/87148630/161849820-39e79c24-4a9d-4e4a-b0c7-501a4604c217.svg)
* If version report is different to version installed show this [icon](https://fonts.google.com/icons?selected=Material%20Icons%20Outlined%3Acode%3A) ![code_black_24dp](https://user-images.githubusercontent.com/87148630/161850217-38885fe2-37e8-4379-ab96-9a0c5e1abb7c.svg)
* If we don't have report for an app show this [icon](https://fonts.google.com/icons?selected=Material%20Icons%20Outlined%3Aclose%3A) with red background ![close_black_24dp](https://user-images.githubusercontent.com/87148630/161850393-cf1c90b7-abab-4862-9767-9aac62ac4a0f.svg)
* And we continue to show text on report page

**Comment by theimpulson** [comment]
Great suggestion. I will try doing it and share what it looks like.

**Comment by Jean-BaptisteC** [comment]
![Screenshot 2022-04-10 at 09-10-44 Figma](https://user-images.githubusercontent.com/87148630/162606780-a9963ed2-b27a-4a77-943a-a26ac33a87cb.png)

For icons, follow theme system
For trackers, change icon and use this  ![visibility_black_24dp](https://user-images.githubusercontent.com/87148630/162607523-b71d5987-33ba-463c-9f97-2289c4ee135e.svg) and show ?, when we have not reports about app
![Screenshot 2022-04-10 at 09-44-16 Figma](https://user-images.githubusercontent.com/87148630/162608025-0454e87a-1e4f-4ba6-9955-f70fa77b6171.png)
![Screenshot 2022-04-10 at 09-45-00 Figma](https://user-images.githubusercontent.com/87148630/162608048-fed59ed0-f00a-4789-98c7-879a2da2f99c.png)

**Comment by Jean-BaptisteC** [comment]
I think solutions to explain significations of icons (done, code and close).

**Comment by Jean-BaptisteC** [comment]
![Screenshot 2022-04-14 at 10-52-21 Figma](https://user-images.githubusercontent.com/87148630/163350050-074a44a4-a824-4d7b-a650-b78506a8cac1.png)
Add this new chip when we have not reports for app (chip follow material you)

**Comment by Jean-BaptisteC** [comment]
Add click on chips (done, code and close)
Show toast or snackbar

Add string `App version = Report version` for this icon ![done_black_24dp](https://user-images.githubusercontent.com/87148630/161849820-39e79c24-4a9d-4e4a-b0c7-501a4604c217.svg)
Replace this [string](https://github.com/Exodus-Privacy/exodus-android-app/blob/51009bed5f8e8e2911d640d8fd16f603443b6280/app/src/main/res/values/strings.xml#L16) by `App version <> Report version` for this icon ![code_black_24dp](https://user-images.githubusercontent.com/87148630/161850217-38885fe2-37e8-4379-ab96-9a0c5e1abb7c.svg)
Replace this [string](https://github.com/Exodus-Privacy/exodus-android-app/blob/51009bed5f8e8e2911d640d8fd16f603443b6280/app/src/main/res/values/strings.xml#L17) by `This app hasn\'t been yet analysed.` for this icon ![close_black_24dp](https://user-images.githubusercontent.com/87148630/161850393-cf1c90b7-abab-4862-9767-9aac62ac4a0f.svg)

Add this feature on App fragment and App detail fragment

**Reply by J-Jamet** [reply]
analyzed with a 'z'

**Comment by Jean-BaptisteC** [comment]
Replace [icon permissions](https://github.com/Exodus-Privacy/exodus-android-app/blob/kotlinRewrite/app/src/main/res/drawable/ic_permissions.xml) by this [icon](https://fonts.google.com/icons?selected=Material%20Icons%20Outlined%3Aperm_device_information%3A)
